File: Python/Easy/Alternating-Digit-Sum.py
class Solution(object):
    def alternateDigitSum(self, n):
        """
        :type n: int
        :rtype: int
        """
        answer = 0
        temp = str(n)
        for i in range(len(temp)):
            answer += int(temp[i]) if i%2==0 else -int(temp[i])
        return answer

File: Java/Easy/Alternating-Digit-Sum.java
class Solution {
    public int alternateDigitSum(int n) {
        int sign = Integer.toString(n).length()%2==0 ? -1:1;
        int answer = 0;
        while (n>0) {
            answer+=sign*(n%10);
            sign*=-1;
            n/=10;
        }
        return answer;
    }
}
